packages/sources/finage/src/endpoint/etf.ts

```diff
@@ -25,24 +29,30 @@ export interface ResponseSchema {
 }
 
 export const makeEtfExecute =
-  (executeConfig: { country?: string }): ExecuteWithConfig<Config> =>
+  (country?: string): ExecuteWithConfig<Config> =>
   async (request, _, config) => {
     const validator = new Validator(request, inputParameters, {}, { overrides })
 
     const jobRunID = validator.validated.id
     const base = validator.validated.data.base
     const symbol = validator.overrideSymbol(NAME, base).toUpperCase()
+    const param_country = validator.validated.data.country
+      ? validator.validated.data.country.toLocaleLowerCase()
+      : country
+      ? country
+      : null
 
     const url = util.buildUrlPath('/last/etf/:symbol', { symbol })
 
     const params = {
       apikey: config.apiKey,
+      country: param_country,
     }
 
     const options = {
       ...config.api,
       url,
-      params: executeConfig.country ? { ...params, country: executeConfig.country } : params,
+      params: params,
```

Comment:
This change means we can now send a request with the param `country` set to `null` - have we tested that the DP API accepts this?

Personally I'd prefer to not include `country` at all if it isn't set, but if the API spec accepts `null` then this would be fine.

Reply:
API accepts null as value for country and generates the correct url.